db/migrate/20220809095921_create_posts.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :users, null:false, foreign_key: true
+      t.string :title
+      t.string :text
+      t.integer :commentscounter
+      t.integer :likescounter
```

Comment:
- [ ] please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that